README.md

```diff
@@ -30,9 +30,30 @@ In FFI (`-m sys`) mode, `gir` generates as much as it can. So in this mode, the
 ignore = ["Gtk.Widget", "Gtk.Window"]
 ```
 
+You also need to add any needed external libraries in the "external_libraries" parameter. Example:
+
+```toml
+external_libraries = [
+   "GLib",
+   "GObject",
+]
+```
+
+You can specify a few other options:
```

Comment:
IMHO also better add "Part of its can overridden from command line"

Reply:
I didn't know, how?

Reply:
Also not sure :(

Reply:
Haha! Then let's not mention it. I'll update the other issues. ;)